components/apimgt/org.wso2.carbon.apimgt.rest.api.core/src/main/java/org/wso2/carbon/apimgt/rest/api/core/impl/ApisApiServiceImpl.java

```diff
@@ -58,4 +63,39 @@ public Response apisApiIdGatewayConfigGet(String apiId, String accept,
                     MediaType.APPLICATION_JSON).entity(errorDTO).build();
         }
     }
+
+    /**
+     *
```

Comment:
We need to have {@inheritDoc} and also explain what this overwrite method will do.